frontend/src/app/shared/components/profile-menu/profile-menu.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="user-info">
+    <div class="avatar"></div>
+    <div class="name-email">
+        <p class="user-name">John Doe</p>
+        <p class="user-email">user@example.com</p>
+    </div>
+</div>
+<hr class="divider">
+<button mat-menu-item class="menu-item">
+    <mat-icon class="icon" aria-hidden="true">
+        <img src="assets/profile_icon.svg" alt="Edit Profile Icon">
+    </mat-icon>
+    Edit Profile
+</button>
+<button mat-menu-item class="menu-item">
+    <mat-icon class="icon" aria-hidden="true">
+        <img src="assets/settings_icon.svg" alt="Settings Icon">
+    </mat-icon>
+    Settings
+</button>
+<button mat-menu-item class="menu-item logout">
+    <mat-icon class="icon logout-icon" aria-hidden="true" style="color: #6f42c1;">
+        <img src="assets/logout_icon.svg" alt="Logout Icon">
+    </mat-icon>
+    Log out
```

Comment:
Can you and log out method that delete access tokens and also redirect to login?